src/AsmResolver.Workspaces.DotNet/DotNetRelations.cs

```diff
@@ -48,9 +48,17 @@ public static class DotNetRelations
             new Guid("ce11d2f6-a423-429d-ad37-2f073fdf63be"));
 
         /// <summary>
-        /// Describes the relationship between a type definition and its reference.
+        /// Describes the relationship between a method and its specification.
         /// </summary>
-        public static readonly ObjectRelation<TypeDefinition, TypeReference> ReferenceType = new(
+        public static readonly ObjectRelation<IMethodDefOrRef, MethodSpecification> ReferenceMethodSpecification = new(
```

Comment:
Do we really need this? What about replacing the `MethodSpecification` with `IMemberDescriptor`, and then just reuse `ReferenceMember`, similar to how we reused the `ReferenceType` for type specs?

Reply:
Yes but in that case we loose support for FieldReferences, and than we would have to create new relation for those.

Reply:
That would be the case if we used `IMethodDescriptor`, but we can use `IMemberDescriptor` instead to capture both.